frontend/src/routes.js

```diff
@@ -181,7 +171,7 @@ class PhishTray extends Component {
 
 export default connect(
   state => ({
-    appCheck: state.exercise.id,
+    isAllowed: state.exercise.id,
```

Comment:
let's check for participant id instead which is given when the exercise is initialised